runtime/runtime_controller.h

```diff
@@ -176,10 +181,17 @@ class RuntimeController : public PlatformConfigurationClient {
   ///             including the implicit view. Adding a view that already exists
   ///             triggers an assertion.
   ///
+  ///             If the isolate is not running, the view will be saved and
+  ///             flushed to the isolate when it starts.
```

Comment:
```suggestion
  ///             If the isolate is not running, the view will be saved and
  ///             flushed to the isolate when it starts, and the callback
  ///             is not completed until then. If such a view is removed 
  ///             before the isolate starts, the callback is completed immediately with 
  ///             false.
```

Reply:
I added this and tweaked the wording a bit. I also added similar content to the `AddViewCallback` declaration too.